lib/engine/share.rb

```diff
@@ -63,6 +66,12 @@ def inspect
       "<Share: #{@corporation.id} #{@percent}%>"
     end
 
+    def preferred_percent
+      return 0 if @preferred
+
+      @percent
+    end
```

Comment:
The naming seems a little strange. This says that preferred_percent is zero if it's preferred?

Shouldn't this be "common_percent" ?

Reply:
I see what you mean by this; that's a bit clearer than what I was thinking (preferred_percent -> percent accounting for preferred status)

Reply:
Done